### ISSUE TITLE
Accent.c: build fails

### ISSUE THREAD

**Issue by andy5995** [andy5995/htdig#1]
Build fails at Accent.c:
```
[...]
 g++ -DHAVE_CONFIG_H -I. -I. -I../include -DDEFAULT_CONFIG_FILE=\"/opt/www/conf/htdig.conf\" -I../include -I../htlib -I../htnet -I../htcommon -I../htword -I../db -I../db -DMV=\"/bin/mv\" -g -O2 -Wall -fno-rtti -fno-exceptions -c Metaphone.cc -o Metaphone.o >/dev/null 2>&1
Accents.cc:78:42: error: narrowing conversion of '128' from 'int' to 'char' inside { } [-Wnarrowing]
    'o', 'u', 'u', 'u', 'u', 'y', 254, 255};
                                          ^
Accents.cc:78:42: error: narrowing conversion of '129' from 'int' to 'char' inside { } [-Wnarrowing]
Accents.cc:78:42: error: narrowing conversion of '130' from 'int' to 'char' inside { } [-Wnarrowing]
Accents.cc:78:42: error: narrowing conversion of '131' from 'int' to 'char' inside { } [-Wnarrowing]
Accents.cc:78:42: error: narrowing conversion of '132' from 'int' to 'char' inside { } [-Wnarrowing]
Accents.cc:78:42: error: narrowing conversion of '133' from 'int' to 'char' inside { } [-Wnarrowing]
[...]
```

Debian's patch for 3.2.0b6-16 may help fix this.

https://packages.debian.org/stable/web/htdig

htdig_3.2.0b6-16.debian.tar.xz

**Comment by andy5995** [comment]
I reverted to a later commit. This issue may no longer be valid

**Comment by andy5995** [comment]
fixed by obsolete as of df02c94a6ad4c56ade8663b709f0b035ebda721d
